buildozer/targets/ios.py

```diff
@@ -242,25 +252,24 @@ def build_package(self):
         self.buildozer.rmdir(intermediate_dir)
 
         self.buildozer.info('Creating archive...')
-        self.xcodebuild((
-                ' -alltargets'
-                ' -configuration {mode}'
-                ' -scheme {scheme}'
-                ' -archivePath "{xcarchive}"'
-                ' archive'
-                ' ENABLE_BITCODE=NO'
-            ).format(mode=mode, xcarchive=xcarchive, scheme=app_name.lower()),
+        self.xcodebuild(
+            '-alltargets',
+            f'-configuration {mode}',
+            f'-scheme {app_name.lower()}',
+            f'-archivePath "{xcarchive}"',
+            'archive',
+            'ENABLE_BITCODE=NO',
+            self.code_signing_development_team,
             cwd=build_dir)
 
         self.buildozer.info('Creating IPA...')
-        self.xcodebuild((
-                ' -exportArchive'
-                ' -exportFormat IPA'
-                ' -archivePath "{xcarchive}"'
-                ' -exportPath "{ipa}"'
-                ' CODE_SIGN_IDENTITY={ioscodesign}'
-                ' ENABLE_BITCODE=NO'
-            ).format(xcarchive=xcarchive, ipa=ipa_tmp, ioscodesign=ioscodesign),
+        self.xcodebuild(
+            '-exportArchive',
+            f'-archivePath "{xcarchive}"',
+            f'-exportOptionsPlist "{plist_rfn}"',
```

Comment:
After removal of `exportFormat` option and without `exportOptionsPlist` option the command was failing with error ```xcodebuild: error: The flag -exportOptionsPlist is required when specifying -exportArchive```.